tests/test_ag3.py

```diff
@@ -851,3 +851,12 @@ def test_xpehh_gwss():
     # check some values
     assert_allclose(x[0], 467448.348)
     assert_allclose(xpehh[:, 2][100], 0.4817561326426265)
+
+
+def test_karyotyping():
+    ag3 = setup_ag3(cohorts_analysis="20230516")
+
+    df = ag3.karyotype(inversion="2La", sample_sets="AG1000G-FR", sample_query=None)
+
+    assert isinstance(df, pd.DataFrame)
+    assert len(df) > 0
```

Comment:
Suggest to also check expected columns are present and expected dtypes of columns. Also expected range or set of values.